homework/english/Alexandr_Konduhovich/lecture1/task1.md

```diff
@@ -0,0 +1,48 @@
+# Homework1 Task1
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum is working today?
+2. Why these boys are is talking now?
+3. Where does he live?
+4. Does their mother cook very well?
+5. Lili is not rich, she does not drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is geting dark.
+8. A young man usually  stands at the window.
+9. Ann usually makes doll dresses herself.
+10. His wife is having coffee at the moment?
+11. Let’s go for a walk. It is not raining.
+12. Look! The boys are coming home.
+13. Maila likes Chinese food?
+14. My grandmother does not  work. She is on pension.
+15. What your brother is watching now?
+16. Where do you live?
+17. Do you  play the piano well?
+18. Where their brother are swiming now?
```

Comment:
Where are their brothers swimming now￼?

Reply:
or Where is their brother swimming now?
is - brother
are - brothers